test/ng/snifferSpec.js

```diff
@@ -25,6 +25,25 @@ describe('$sniffer', function() {
     });
 
 
+    it('should be true if running inside nw.js', function() {
```

Comment:
As the intend of this PR was to allow history in NW apps, I'd say it's a good idea to have a test that reflects that change and test it (despite the actual technical implementation).

Incorporate this into the test for Chromed Packaged App sounds odd to me, as NW apps are not Chromed Packaged Apps (correct me if I'm wrong here).

Apart from that, I'll be happy to change the test and try to incorporate it into the one you're talking about.